content/docs/error-boundaries.md

```diff
@@ -142,25 +143,25 @@ class MyComponent extends React.Component {
 
   handleClick() {
     try {
-      // Do something that could throw
+      // Hata oluşturacak bir şey yapın
     } catch (error) {
       this.setState({ error });
     }
   }
 
   render() {
     if (this.state.error) {
-      return <h1>Caught an error.</h1>
+      return <h1>Hata yakalandı.</h1>
     }
-    return <div onClick={this.handleClick}>Click Me</div>
+    return <div onClick={this.handleClick}>Beni tıkla</div>
   }
 }
 ```
 
-Note that the above example is demonstrating regular JavaScript behavior and doesn't use error boundaries.
+Yukarıdaki örnekte normal JavaScript davranışlarının gösterildiğini ve hata sınırlarının kullanılmadığını dikkate alınız.
 
-## Naming Changes from React 15 {#naming-changes-from-react-15}
+## React 15'ten Sonraki İsim Değişiklikleri {#naming-changes-from-react-15}
 
-React 15 included a very limited support for error boundaries under a different method name: `unstable_handleError`. This method no longer works, and you will need to change it to `componentDidCatch` in your code starting from the first 16 beta release.
+React 15, hata sınırlarını çok limitli bir şekilde destekleyen, `unstable_handleError` isimli başka bir metod içermekteydi. Bu metod artık çalışmıyor ve onu ilk 16 beta versiyonundan itibaren kodunuzda `componentDidCatch` ile değiştirmeniz gerekmektedir.
 
-For this change, we’ve provided a [codemod](https://github.com/reactjs/react-codemod#error-boundaries) to automatically migrate your code.
+Bu değişiklik için, kodunuzdaki göçü otomatikleştirmek adına bir [codemod](https://github.com/reactjs/react-codemod#error-boundaries) sunuyoruz.
```

Comment:
What about `kodunuzdaki yukseltmeyi` ?

Reply:
I'm definitely open for suggestions here (it doesn't sound right, I know), but `yükseltme` sounds like an upgrade to me. Migration is not necesserily an upgrade, in my opinion.

What about `taşımak`? Something like

`...kodunuzun taşınmasını otomatikleştirmek adına...`

Reply:
`tasinma` sounds great! Let's use it.